src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5908,6 +5908,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeReserved" xml:space="preserve">
     <value>The type name '{0}' is reserved to be used by the compiler.</value>
   </data>
+  <data name="ERR_ReservedTypeMustFollowPattern" xml:space="preserve">
```

Comment:
>ERR_ReservedTypeMustFollowPattern [](http://example.com/codeflow?start=14&length=33)

Do we expect to use this error for other types? (The message is quite specific.) If not, consider giving the error a more specific name, and perhaps hardcoding the type name in the message.